src/main/java/fi/helsinki/cs/tmc/cli/backend/Settings.java

```diff
@@ -92,7 +102,7 @@ public String getFormattedUserData() {
         if (!userDataExists()) {
             return "";
         }
-        return getUsername() + ":" + this.getPassword();
+        return getUsername().get() + ":" + this.getPassword().get();
```

Comment:
What's the purpose of this? Isn't the password supposed to be almost always absent in the oauth branch?

Reply:
It's true that password is actually always `null` in this branch. However to me it is unclear where the method is used in general (it is a part of the Settings interface) , so I thought that the point of the method is to get the password as it returned it before.